charts/osbuilder/templates/deployment.yaml

```diff
@@ -87,4 +91,36 @@ spec:
             {{- with .Values.tolerations }}
             tolerations:
             {{- toYaml . | nindent 14 }}
-            {{- end }}
\ No newline at end of file
+            {{- end }}
+---
```

Comment:
just a note for the future, we might want to make this configurable and not provide our own http service here. we should have follow-ups

Reply:
agreed